sktime/datasets/_single_problem_loaders.py

```diff
@@ -13,7 +13,7 @@
     "jasonlines",
     "achieveordie",
     "ciaran-g",
-    "jonathanbechtel",
+    "jonathanbechtel"
```

Comment:
these commas should remain, in accordance with code formatting requirements. Is your local linter setting automatically removing them? See here how to set up linting locally so it agrees with the conventions:
https://www.sktime.net/en/stable/developer_guide/coding_standards.html#setting-up-local-code-quality-checks

Reply:
fixed this now